src/healthCheck/HealthCheck.js

```diff
@@ -88,7 +88,7 @@ class HealthCheck {
    * @description Gets the health of the service along with sub-services
    *
    */
-  async getHealth () {
+  async getHealth (context = undefined) {
```

Comment:
what for make default value be `undefined`? Can't we just have code like this:
```
async getHealth (context) {
```

Reply:
That makes it obvious that the parameter is optional without looking into the implementation.